lib/typescript/components/general/AudioClip/index.tsx

```diff
@@ -108,8 +107,19 @@ export const AudioClip = ({audioUrl}: AudioRenderProps) => {
     if (audioElement.current.currentTime === audioElement.current.duration) {
       colorPlaybackBarsWhite(19, 0, canvasContext, barsSamplesPaths, setCount);
     }
-    audioElement.current.play();
-    setIsPlaying(true);
+
+    const playPromise = audioElement.current.play();
+
+    if (playPromise) {
+      playPromise
+        .then(() => {
+          audioElement.current.play();
+          setIsPlaying(true);
+        })
+        .catch(() => {
+          setError(true);
+        });
+    }
```

Comment:
this handle errors in case the audio file is broken (for ex. no supported source found): in this case, the component displays 'could not load audio'